clash-lib/src/Clash/Core/Type.hs

```diff
@@ -104,7 +104,7 @@ import           Clash.Core.Name
 import {-# SOURCE #-} Clash.Core.Subst
 import           Clash.Core.TyCon
 import           Clash.Core.Var
-import           Clash.Unique
+import qualified Clash.Data.UniqMap as UniqMap (find, lookup)
```

Comment:
Because they're the two functions that were used in this file already? `find` is a most easily-distinguishable renaming of the previous `lookupUniqMap'`, where the prime means _throw an error if this isn't in the map_